simulation/simulation_interface/proto/geometry_msgs.proto

```diff
@@ -1,58 +1,65 @@
 syntax = "proto3";
 
 /**
- * Protobuf package to define geometry_msgs in ROS2.(https://github.com/ros2/common_interfaces/tree/master/geometry_msgs)
-*/
+ * Protobuf package to define geometry_msgs in
+ * ROS2.(https://github.com/ros2/common_interfaces/tree/master/geometry_msgs)
+ */
 package geometry_msgs;
 
 /**
- * Protobuf definition of [geometry_msgs/msg/Point type in ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Point.msg)
-**/
+ * Protobuf definition of [geometry_msgs/msg/Point type in
+ *ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Point.msg)
+ **/
 message Point {
-    double x = 1; // X value in a cartesian coordinate.
-    double y = 2; // Y value in a cartesian coordinate.
-    double z = 3; // Z value in a cartesian coordinate.
+  double x = 1; // X value in a cartesian coordinate.
+  double y = 2; // Y value in a cartesian coordinate.
+  double z = 3; // Z value in a cartesian coordinate.
 }
 
 /**
- * Protobuf definition of [geometry_msgs/msg/Quaternion type in ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Quaternion.msg)
-**/
+ * Protobuf definition of [geometry_msgs/msg/Quaternion type in
+ *ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Quaternion.msg)
+ **/
 message Quaternion {
-    double x = 1; // X value in a quaternion. (0 <= x <= 1)
-    double y = 2; // Y value in a quaternion. (0 <= y <= 1)
-    double z = 3; // Z value in a quaternion. (0 <= z <= 1)
-    double w = 4; // W value in a quaternion. (0 <= w <= 1)
+  double x = 1; // X value in a quaternion. (0 <= x <= 1)
+  double y = 2; // Y value in a quaternion. (0 <= y <= 1)
+  double z = 3; // Z value in a quaternion. (0 <= z <= 1)
+  double w = 4; // W value in a quaternion. (0 <= w <= 1)
 }
 
 /**
- * Protobuf definition of [geometry_msgs/msg/Pose type in ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Pose.msg)
-**/
+ * Protobuf definition of [geometry_msgs/msg/Pose type in
+ *ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Pose.msg)
+ **/
```

Comment:
Malformed indentation

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/357/commits/d05e45028e960780dfb581d3dd15573f3f18f803